### DISCUSSION TITLE
my tailwind doesnt work on my react app

### DISCUSSION THREAD

**Discussion by Moontaz** [tailwindlabs/tailwindcss#13091] [Help]
i have react app that use tailwind and nothing wrong when i use localhost. until i deploy it to github and my setting doesnt work att all
[https://github.com/Moontaz/test-ui](url) u can see my file there. please help!

**Comment by wongjn** [comment]
You can see on the deployed page, the CSS has not been processed by Tailwind (notice the `@tailwind` directives):

```css
@tailwind base;@tailwind components;@tailwind utilities;…
```

This is because in your [GitHub pages deployment](https://github.com/Moontaz/test-ui/actions/runs/8161515086/job/22310462236), you are using the default configuration that builds with Jekyll. However, from the structure of the repository, it seems you'd want to build with Node.js. Consider configuring GitHub pages to build your project with Node.js-based commands instead. Refer to [the documentation on GitHub pages](https://docs.github.com/en/pages) for more details, and perhaps seeking assistance in [the GitHub pages discussion area](https://github.com/orgs/community/discussions/categories/pages) if you are still stuck.

**Reply by Moontaz** [reply]
i still cant understand. but actually i make react with tailwind and typescrypt also because i just using template so i sont know how to deploy it
